src/com/jalasoft/search_files/utils/Search_Validation.py

```diff
@@ -0,0 +1,54 @@
+import re
+import bitmath
+
+
+class SearchValidation(object):
+    def __init__(self):
+        pass
+
+    def string1_exists_within_string2(self, string_1, string_2, flag):
+        case_sensitive = bool(flag)
+        if case_sensitive is True:
+            search_match = re.search(string_1, string_2, re.IGNORECASE)
+            if search_match is None:
+                return False
```

Comment:
(1): Those functions have more than one return and it can be inconsistent with other functions. Do you have an argument that it can optimize our python program?

Reply:
this method works well, but my possible task will be to simplify it for the next sprint